installation/README.md

```diff
@@ -1,11 +1,9 @@
 # Installation
 
-Some simple guides to setting up the software on your Raspberry Pi.
+We recommend using [Raspberry Pi Imager](https://www.raspberrypi.org/software/) to install an operating system on your Raspberry Pi. As well as Raspberry Pi OS, you can use it to install other operating systems such as Ubuntu or LibreELEC (Kodi media centre).
```

Comment:
Should probably make the "Raspberry Pi OS" text a link to `../raspbian/README.md` ?

Reply:
No - that would complicate it. Better to just link straight to the download page, as I have done with the Imager link.